consume.go

```diff
@@ -353,9 +353,9 @@ func (cmd *consumeCmd) encodeAvro(data []byte) (json.RawMessage, error) {
 	if err != nil {
 		return nil, fmt.Errorf("cannot get schema for id %d: %v", id, err)
 	}
-	codec, err := goavro.NewCodec(schema.String())
+	codec, err := goavro.NewCodec(schema.CanonicalString(0))
```

Comment:
Maybe add a comment linking to https://github.com/linkedin/goavro/issues/198 ?

    // Canonicalize the schema to remove default values and logical types
    // to work around https://github.com/linkedin/goavro/issues/198